### DISCUSSION TITLE
pdfpages (eso-pic.sty & "old" LaTeX release)

### DISCUSSION THREAD

**Discussion by c-kloukinas** [MiKTeX/miktex#853] [Q&A]
Hi all,
I'm using MikTeX on MacOS and trying to use pdfpages.
The processing breaks because of eso-pic.sty - it requires a newer release of LaTeX than the one MikTeX offers:
```
"/Library/Application Support/MiKTeX/texmfs/install/tex/latex/eso-pic/eso-pic.sty"

LaTeX Warning: You have requested release `2020-10-01' of LaTeX,
               but only release `2020-02-02' is available.

! Undefined control sequence.
<argument> \nointerlineskip \AddToHook 
                                       {shipout/background}{\put (0,\ESO@yof...
l.188 }
       
? 
```

Not sure what to do with this - I'd expect all packages offered by the system to use a LaTeX release that is at most the one offered by MikTeX.

My MikTeX console is v 2.9.7364
It's installed in administrator mode, if this is of any use. Attaching file miktex-report.txt as well.
Best,
Chris
P.S. A separate and most probably irrelevant matter - miktex-misc is broken. Trying to update it and it always shows up as requiring an update. (saw a ticket from someone on Windows - so it's the same on MacOS).
[miktex-report.txt](https://github.com/MiKTeX/miktex/files/6708187/miktex-report.txt)

P.P.S. It'd be amazing if there was an easy way to export the list of packages installed in a system and import it in another system, so as to be able to easily transfer one's local MikTeX installation to another system and/or back it up.

**Comment by edocevoli** [comment]
Indeed, it seems that your LaTeX base package is outdated. Running the updater 3 times (as user, as admin, as user) should give you the latest package.
